<launch>
	<node pkg="darknet_ros" type="darknet_ros" name="darknet" output="screen"></node>

</launch>
